### PR TITLE
Link

### PR DESCRIPTION
**Pull request by jackcarlisle**

### CODE REVIEW

File: src/actions/tag.js [hunk 1]
```diff
@@ -11,47 +11,42 @@ export function search () {
...
-  };
-}
-
-/*
 * onChangeText function for the search input component
 */
-export function setSearchTerm (text) {
-  return { type: types.SET_SEARCH_TERM, text };
+export function setSearchString (text) {
+  console.log(':::::::;;;;;;::::::;;;;;;;', text);
```
Comment (jruts) [inline src/actions/tag.js]: might want to clean these up a bit :) :baby_chick:

File: src/actions/tag.js [hunk 2]
```diff
@@ -11,47 +11,42 @@ export function search () {
...
-    return graphqlService.query(QUERY_SEARCH_TAGS, {searchTerm})
+export function fetchTags (searchString, start, size) {
+  console.log('fetchTags >>>>>>>', searchString, start, size);
+  return (dispatch, state) => {
+    console.log('STATE', state());
+    return graphqlService.query(QUERY_SEARCH_TAGS, {id: searchString, start, size})
       .then(json => {
-        const tags = json.data.taggable;
-        return dispatch(setSearchResults(tags));
+        console.log('*****JSON*****', json);
```
Comment (jruts) [inline src/actions/tag.js]: and this :hamburger:

File: src/components/search-bar/index.js [hunk 3]
```diff
@@ -5,50 +5,66 @@
 */
 
 import React, { PropTypes, Component } from 'react';
-
 import { Input } from 'react-bootstrap';
+// import { SymbolButton as Button } from '../button';
```
Comment (jruts) [inline src/components/search-bar/index.js]: If we don't use it, remove it =>  :bath: :man:

**Comment by jackcarlisle** [comment]
Bathman

**Pull request merged**